### ISSUE TITLE
Non sentence "xu"

### ISSUE THREAD

**Issue by rTreutlein** [rTreutlein/opencog#3]
mi xu jimpe
is it me who understands

```
SatisfactionLink
  InherintanceLink
     ConceptNode "I"
     SatisfyingSetLink
         VarNode "1"
         ExistsLink
             VarNode "pred"
             ListLink
                  EvaluationLink
                      VarNode "pred"
                      VarNode "I"
                 ImplicationLink
                      VarNode "pred"
                      PredicateNode "understand"
```

**Comment by rTreutlein** [comment]
I have been thinking a bit more and think that the above is not the right way to solve this.

First of all the Question itself should be simplified to the usual:
```
SatisfactionLink
   EvaluationLink
      PredicateNode "jimpe_123"
      ConceptNode "mi_456"
```
Additionally we want to say that there exists something that applies to "jimpe_123"
```
ExistsLink
   VarNode "Concept"
   EvaluationLink
      PredicateNode "jimpe_123"
      VarNode "Concept"
```

This not only expresses exactly what we want but is also significantly easier to create in the parser.

**Comment by rTreutlein** [comment]
Implemented in c43c21bbea929297be39225123b9d081a8683e32
